lib/foreman_rh_cloud/engine.rb

```diff
@@ -151,6 +152,12 @@ def self.register_scheduled_task(task_class, cronline)
       end
     end
 
+    initializer "foreman_rh_cloud.add_rabl_view_path" do
```

Comment:
Is this what this is doing "This included adding RABL infrastructure to foreman_rh_cloud, since it didn't use rabl previously"

Reply:
Yes, bingo